src/Sentry.Unity/WebGL/SentryWebGL.cs

```diff
@@ -36,6 +37,9 @@ public static void Configure(SentryUnityOptions options)
                 options.DiagnosticLogger?.LogWarning("Attaching screenshots on WebGL is disabled - " +
                     "it currently produces blank screenshots mid-frame.");
             }
+
+            // Use AnalyticsSessionInfo.userId as the default UserID in native & dotnet
+            options.DefaultUserId = AnalyticsSessionInfo.userId;
```

Comment:
If this is an ID that can be used to link the user across apps/services, we shouldn't be using it here.
We purposely on some GUIDs we create so that we can have some consistency in detecting "number ofusers impacted" but the goal is never to do actual user tracking. For that reason we've dropped "Android ID" also, for example

Reply:
JS will default to the connection IP Address, and there's no native so we might be better off without anything here

Reply:
1. we don't have sentry-js (yet)
2. therefore setting this here only affects c# events
3. I'm setting it _here_ because that's what feeled right - each "native" integration defines the source for the DefaultUserId value
4. > If this is an ID that can be used to link the user across apps/services, we shouldn't be using it here.

    I really don't think so. It's an installation ID.
    
   > https://docs.unity3d.com/ScriptReference/Analytics.AnalyticsSessionInfo-userId.html 
   > A random GUID uniquely identifying sessions played on the same instance of your game or app.
   > Unity generates the userId GUID immediately after installation. The userId value is only regenerated if the user re-installs the game.